app/views/holidays/_form.html.erb

```diff
@@ -53,7 +53,7 @@
     <%= f.label :replacement, class: 'control-label' %>
     <%# Capybara doesn't seem to find invisible selection boxes, so this div has to be displayed in a test environment %>
     <%= content_tag :div, class: "controls", id: 'selection', style: "#{Rails.env}" == 'test' ? "display:block" : "display: none" do %>
-      <%= f.collection_select :replacement_user_id, User.all.select { |user| user.chair == current_user.chair && user.is_wimi? }, :id, :name, include_blank: true, id: 'replacement_selection' %>
```

Comment:
Könnte man hier nicht den ersten in der Liste als Default einsetzen? Denn wenn man ja den Haken setzt will man doch nicht niemanden auswählen (können)... Außerdem ist es schöner ;)